src/QueryType/Update/Query/Command/RawXML.php

```diff
@@ -0,0 +1,83 @@
+<?php
+
+namespace Solarium\QueryType\Update\Query\Command;
+
+use Solarium\QueryType\Update\Query\Query as UpdateQuery;
+
+/**
+ * Update query raw XML command.
+ *
+ * @see http://wiki.apache.org/solr/UpdateXmlMessages
+ */
+class RawXML extends AbstractCommand
```

Comment:
Just for consistency we should camel case XML => RawXml

Reply:
🐫!